modules/developers-guide/examples/bash/start-sg-ccm.sh

```diff
@@ -0,0 +1,16 @@
+ccm create stargate -v 3.11.8 -n 1 -s -b
```

Comment:
I've put up a PR to update the readme https://github.com/stargate/stargate-grpc-rust-client/pull/15. I think it would be good if we just used docker across the board since it'll simplify things.

Reply:
I've changed the docs.